src/FSharp.Data.DesignTime/FSharp.Data.DesignTime.fsproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\netfx.props" />
+  <PropertyGroup>
+    <TargetFrameworks>net45; netstandard2.0</TargetFrameworks>
+    <DefineConstants>NO_GENERATIVE</DefineConstants>
```

Comment:
Is this still needed with the latest TPSDK?

Reply:
Since this is an erasing type provider, using this define makes the design-time DLL smaller.  

But yes, we could now optionally make this a generative type provider.  That would be separate thing and I think would have to be revealed as a separately named TP, since whether a TP is generative or not is determined before the static parameters are applied:
```fsharp
type MyXml = XmlProviderGenerative<"foo.xml">
```
Besides I believe it would be easy (it may stilll uncover bugs in generative TP support, though that's pretty solid - Swagger has ironed out the issues).